code/modules/projectiles/projectile/bullets.dm

```diff
@@ -112,6 +112,27 @@
 /obj/item/projectile/bullet/midbullet3
 	damage = 30
 
+/obj/item/projectile/bullet/midbullet3/silencer
+	damage = 30
+
+/obj/item/projectile/bullet/midbullet3/silencer/on_hit(atom/target, blocked = 0)
+	if(..(target, blocked))
+		var/mob/living/M = target
+		M.silent += 1

```

Comment:
status effects with a value of 1 generally aren't going to be very good because of how `Life` ticks.

Also, high damage+silence on the same bullet on a spammable weapon...not exactly fair or balanced.